diff --git a/dist/index.mjs b/dist/index.mjs
index a9541e0f283ee7d12a82962d76916cfd09b95d16..c30c84fb703d5f741177e30fa877f7fafade2114 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -989,11 +989,13 @@ function emitLegacyIndexDeprecation(url, packageJsonUrl, base, main) {
  * @returns {Stats}
  */
 function tryStatSync(path) {
-  // Note: from Node 15 onwards we can use `throwIfNoEntry: false` instead.
   try {
     return statSync(path)
   } catch {
-    return new Stats()
+    const stub = Object.create(null)
+    stub.isFile = () => false
+    stub.isDirectory = () => false
+    return stub
   }
 }
 
