diff --git a/dist/shared/nuxi.1902c37d.mjs b/dist/shared/nuxi.1902c37d.mjs
index 70441f6b2bf94915cc87a158067af5b0004dc326..7ce0750e7c6b0a5b46e093da94a8656b8a3415d8 100644
--- a/dist/shared/nuxi.1902c37d.mjs
+++ b/dist/shared/nuxi.1902c37d.mjs
@@ -6909,11 +6909,13 @@ function emitLegacyIndexDeprecation(url, packageJsonUrl, base, main) {
  * @returns {Stats}
  */
 function tryStatSync(path) {
-  // Note: from Node 15 onwards we can use `throwIfNoEntry: false` instead.
   try {
     return statSync(path)
   } catch {
-    return new Stats()
+    const stub = Object.create(null)
+    stub.isFile = () => false
+    stub.isDirectory = () => false
+    return stub
   }
 }
 
